CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.25.3-dev
+* Bug - Update router to clean url with multi-slash, it will redirect "//path///to/handler" to "/path/to/handler" [#1844](https://github.com/aws/amazon-ecs-agent/pull/1844)
```

Comment:
I would prefer having it more descriptive for our customers. 
eg: `Fixed a bug where agent no longer redirected credentials URL` or something similar..

Reply:
updated:)